### DISCUSSION TITLE
How can i get the remote ip im connected to?

### DISCUSSION THREAD

**Discussion by smurfster** [qdm12/gluetun#1878] [Q&A] [answered]
Hey all,

I need to obtain the remote ip, I'm connected to, is there a way I can get this info. Im using torguard with openvpn protocol.

Cheers

**Comment by frepke** [comment, marked as answer]
Check the Wiki: [https://github.com/qdm12/gluetun-wiki/blob/main/setup/test-your-setup.md#check-your-ip-address](https://github.com/qdm12/gluetun-wiki/blob/main/setup/test-your-setup.md#check-your-ip-address)

Think that's what you wanna know 😄 

or `docker exec gluetun wget https://ipinfo.io -O - -q ; echo` in the CLI
